backend/internal/models/review.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "time"
+
+type Review struct {
+	UserID    string    `json:"user_id"`
+	MediaType string    `json:"media_type"`
+	MediaID   string    `json:"media_id"`
+	Rating    string    `json:"rating"`
+	Desc      *string   `json:"desc,omitempty"`
```

Comment:
i think we are going with comment instead of description. i think thats what the column is named in the db